docs/src/main/asciidoc/_developer-guide/Jakarta_Transactions_Reference.adoc

```diff
@@ -196,10 +196,51 @@ the container is responsible to manage transactions. The boundary of the transac
 is defined by the start and end of the method and you influence the behaviour by using
 https://javaee.github.io/javaee-spec/javadocs/javax/ejb/TransactionAttribute.html[`@TransactionAttribute`].
 
-If `java.lang.RuntimeException` is thrown the transaction is rolled back.
-https://javaee.github.io/javaee-spec/javadocs/javax/ejb/EJBContext.html[EJBContext]
-could be used to define the transaction should be rolled-back
-by the end of the method when `setRollbackOnly` is used.
+A way to rollback the transaction is to throw `java.lang.RuntimeException`
+out of the method or use `setRollbackOnly()` method (see <<rolling-back>>).
+
+NOTE: It's not allowed to work with `UserTransaction` within the scope
+      of container managed transaction, and it's neither permitted to
+      ask for the instance with
+      https://javaee.github.io/javaee-spec/javadocs/javax/ejb/EJBContext.html#getUserTransaction--[`EJBContext.getUserTransaction()`].
+      You receive a `RuntimeException` in such case.
+
+==== Bean managed transactions
+
+Using the `@TransactionManagement(TransactionManagementType.BEAN)` means
+the transaction will be managed manually with the use of the JTA API.
+That's with the `UserTransaction` injections and methods on it.
+
+NOTE: You can use injection of `EJBContext` (with `@Resource` annotation)
+      to access the `UserTransaction` instance.
+
+NOTE: If a call is made from the container managed method to bean managed method
```

Comment:
A few points:

Are you referring to EJB calls to another EJB.
What about the other way around.
Why is this different from calling an unannotated EJB method. The reference here to "bean managed methods" seems irrelevant since the comment would apply to calling any EJB method.

The text does not aid my understanding of EJBs and transactions, on the contrary it makes me think I am missing some subtle aspects of the API, it leaves too many unanswered questions.

Reply:
I am going to finish my review at this point (ie I haven't read the rest of the document) and I will put my summary on the final review.

Reply:
I changed the text to use term EJB. I agree it's better. I'm not sure about my phrasing but this is just a note to warn you about some corner cases. This is not about to explain all details behind.

`Why is this different from calling an unannotated EJB method.`) because it's different. Calling from EJB container managed transaction method to EJB container managed transaction method is handled in different manner than calling EJB container managed transaction method to EJB bean managed transaction method.

If you feel I haven't explained some important parts then please provide me with a text and I will be happy to add it.
Or better when this PR is merged you can open new PR where you can explain the omitted parts.